R/bump-version.R

```diff
@@ -24,6 +24,10 @@ bump_version_impl <- function(which) {
 }
 
 get_main_branch <- function() {
+  if (in_example) {
```

Comment:
In the example repos there are not necessarily remotes.
The git config does not help us either because if it insists on master we move the branch to main.

Reply:
it might even make more sense to change the rule for `get_main_branch()` in all cases without remotes, with some warning if there are both "main" and "master" branches?

Reply:
(but accepting the current branch as valid if it's either main or master)

Reply:
Thanks, good points. Can we set up the example repo with a remote pointing to a bare repo in a parallel directory?

Reply:
I've just thought of another workaround, setting the git option to "main" as that's what `get_main_branch()` looks at. (it seemed simpler than setting up the remote)